### DISCUSSION TITLE
cant upgrade ubuntu to chia 1.1.7

### DISCUSSION THREAD

**Discussion by rizlah** [Chia-Network/chia-blockchain#6496] [Support]
tried to follow the upgrade instruction and landed back on 1.1.6
deleted chia-blockchain directory
complete re-install of chia landed me back on 1.1.6 again

anyone having the same issue?

**Comment by rizlah** [comment]
> tried to follow the upgrade instruction and landed back on 1.1.6
> deleted chia-blockchain directory
> complete re-install of chia landed me back on 1.1.6 again
> 
> anyone having the same issue?

in hindsight the version 1.1.7 could just be for windows.

**Comment by rizlah** [comment]
For anyone coming across this post

there is a 1.1.7 for ubuntu.  here are the steps to get it working

close all demons
cd chia-blockchain
. ./activate
chia stop -d all
"pip install chia-blockchain==1.1.7"
chia version

then start chia as normal

**Comment by BrandtH22** [comment]
@rizlah , based on your previous message it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
